scripts/REALTIME.sh

```
@@ -14,7 +14,7 @@ cd cligen
 python proctor_tile_edit.py -s 0 --date=$(date --date '16 hours ago' +'%Y-%m-%d') || exit
 
 cd ../RT
-python enqueue_jobs.py -s 0 && python env2database.py -s 0 --date $(date --date '16 hours ago' +'%Y-%m-%d') && python spam_twitter.py
+python enqueue_wepp_jobs.py -s 0 && python env2database.py -s 0 --date $(date --date '16 hours ago' +'%Y-%m-%d') && python spam_twitter.py
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)